### PR TITLE
Release whiteboard-hypercard v0.3.7

### LINKED ISSUE #729: whiteboard-hypercard build is broken
**Describe the issue**
As part of #700 I attempted to retrigger the build of the whiteboard-hypercard package. The build keeps failing due to a timeout when trying to fetch one of the git submodules. This will need to be resolved separately.

**To Reproduce**
1. Open a PR where the version number for the whiteboard-hypercard package is incremented
2. The build will error

**Expected behaviour**
The package should build with no error.

**Upstream issue**
N/A

**Screenshots**
N/A

**Device/Package information (please complete the following information):**
N/A

**Additional context**
```
[    INFO] toltec.builder: whiteboard-hypercard [rmall]: Fetching source files
[    INFO] toltec.builder: whiteboard-hypercard [rmall]: Building artifacts
[    INFO] toltec.builder: whiteboard-hypercard [rmall]: Only showing up to 50 lines of context. Use --verbose for the full output.
[   ERROR] toltec.builder: whiteboard-hypercard [rmall]: info: downloading component 'rustfmt'
[   ERROR] toltec.builder: whiteboard-hypercard [rmall]: info: installing component 'rustfmt'
[   ERROR] toltec.builder: whiteboard-hypercard [rmall]: Updating crates.io index
[   ERROR] toltec.builder: whiteboard-hypercard [rmall]: Updating git repository `[https://github.com/canselcik/libremarkable`](https://github.com/canselcik/libremarkable%60)
[   ERROR] toltec.builder: whiteboard-hypercard [rmall]: Updating git submodule `git://github.com/ricardoquesada/libfreetype`
[   ERROR] toltec.builder: whiteboard-hypercard [rmall]: warning: spurious network error (2 tries remaining): failed to connect to github.com: Connection timed out; class=Os (2)
[   ERROR] toltec.builder: whiteboard-hypercard [rmall]: warning: spurious network error (1 tries remaining): failed to connect to github.com: Connection timed out; class=Os (2)
[   ERROR] toltec.builder: whiteboard-hypercard [rmall]: error: failed to get `libremarkable` as a dependency of package `marauder v0.3.4 (/src/marauder)`
[   ERROR] toltec.builder: whiteboard-hypercard [rmall]: 
[   ERROR] toltec.builder: whiteboard-hypercard [rmall]: Caused by:
[   ERROR] toltec.builder: whiteboard-hypercard [rmall]: failed to load source for dependency `libremarkable`
[   ERROR] toltec.builder: whiteboard-hypercard [rmall]: 
[   ERROR] toltec.builder: whiteboard-hypercard [rmall]: Caused by:
[   ERROR] toltec.builder: whiteboard-hypercard [rmall]: Unable to update https://github.com/canselcik/libremarkable?branch=master#6e5ce150
[   ERROR] toltec.builder: whiteboard-hypercard [rmall]: 
[   ERROR] toltec.builder: whiteboard-hypercard [rmall]: Caused by:
[   ERROR] toltec.builder: whiteboard-hypercard [rmall]: failed to update submodule `legacy-c-impl/libfreetype`
[   ERROR] toltec.builder: whiteboard-hypercard [rmall]: 
[   ERROR] toltec.builder: whiteboard-hypercard [rmall]: Caused by:
[   ERROR] toltec.builder: whiteboard-hypercard [rmall]: failed to fetch submodule `legacy-c-impl/libfreetype` from git://github.com/ricardoquesada/libfreetype
[   ERROR] toltec.builder: whiteboard-hypercard [rmall]: 
[   ERROR] toltec.builder: whiteboard-hypercard [rmall]: Caused by:
[   ERROR] toltec.builder: whiteboard-hypercard [rmall]: failed to connect to github.com: Connection timed out; class=Os (2)
[   ERROR] toltec.builder: whiteboard-hypercard [rmall]: build() failed
```

### PR DESCRIPTION
**Pull request by fenollp**
Also fixes https://github.com/toltec-dev/toltec/issues/729

Changes since last published version:
* https://github.com/fenollp/reMarkable-tools/releases/tag/v0.3.7
* https://github.com/fenollp/reMarkable-tools/releases/tag/v0.3.6
* https://github.com/fenollp/reMarkable-tools/releases/tag/v0.3.5

### CODE REVIEW

**Comment by fenollp** [comment]
@Eeems Ready! What needs doing here?

File: package/whiteboard-hypercard/package
Comment (Eeems) [inline package/whiteboard-hypercard/package:12]: I didn't realize the code had been relicensed? This is up to you of course, but I would recommend looking closely at the impacts of creative commons on code in all jurisdictions. I seem to recall there being some impacts in germany or something.

File: package/whiteboard-hypercard/package
Comment (Eeems) [inline package/whiteboard-hypercard/package:27]: I wouldn't recommend using pushd/popd here for a single directory change. This is just a recommendation though, I'm still going to approve the PR after it's tested.

**Comment by Eeems** [comment]
> @Eeems Ready! What needs doing here?

A maintainer (likely myself) will need to test this package and then approve and merge it.

**Review by Eeems — APPROVED** [review]

**Pull request merged**
